clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -14613,3 +14613,28 @@ void TypeRemoveRule::runRule(
 }
 
 REGISTER_RULE(TypeRemoveRule, PassKind::PK_Analysis)
+
+void CompatNVCCRule::registerMatcher(ast_matchers::MatchFinder &MF) {
```

Comment:
how about change the rule to: CompatWithClang,  add some comment to detail on diff b/w EDG front end and Clang.

Reply:
Done